Automattic/jetpack
# Assistant: Quick language update

This is a simple one. 

Per a suggestion from @pyronaur, I'm updating the skip CTA on each step of the assistant from "Decide later" to "Not now"

> I’m someone who knows that I don’t want a certain feature, – I’m offered 2 options – “Set it up” or “Decide Later”. Neither of those two options align with what I want – which is not to enable it. Decide later makes it feel like I have to make “the right decision (to enable it)” later, which seems a bit forceful.

> “Not right now” is a slight shift in wording, it still communicates the fact that this can be turned on later, but also doesn’t imply that I will have to make that decision later.

**Before**
![Screen Shot 2021-03-10 at 6 27 44 PM](https://user-images.githubusercontent.com/214813/110711939-61dc3680-81ce-11eb-8113-ee904063a712.png)


**After**
![Screen Shot 2021-03-10 at 6 27 58 PM](https://user-images.githubusercontent.com/214813/110711956-67398100-81ce-11eb-9cbf-e58f76e50d9e.png)

I know this is a massive PR and I should have broken it up into more than one ;-P


#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Go to Jetpack > Dashboard > Recommendations.
* Start the recommendation flow.
* Check link text.
